package.json

```diff
@@ -20,17 +20,19 @@
     "@ar-js-org/ar.js": "^3.4.5",
     "@ar-js-org/artoolkit5-js": "^0.3.0",
     "@artcom/react-three-arjs": "^0.5.6",
+    "@chakra-ui/icons": "^2.1.1",
```

Comment:
Why was this added? I'm not seeing where it is used

Reply:
I was initially using it but ill remove it